recipes/zarp/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "zarp" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 0defdec6fe8dabf66d40f2f4d16dfc2619f509f11763356908af3be51ca049ab
+
+build:
+  noarch: python
+  entry_points:
+   - zarp = zarp.cli:main
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools-git >=1.2
+    - pip
+  run:
+    - addict >=2.4.0
+    - bidict >=0.22.0
+    - email-validator >=1.2.1
+    - jsonref >=0.2
+    - pandas >=1.3.5
+    - pydantic >=1.9.2
+    - pydantic-collections >=0.3.0
+    - pygments >=2.8.0
+    - python >=3.7
+    - rich >=12.5.1
+    - snakemake >=7.19.1
+test:
+  imports:
+    - zarp
+  commands:
+    - zarp --help
+
+
+about:
+  home: https://github.com/zavolanlab/zarp-cli
+  license: Apache License 2.0
+  license_family: APACHE 
+  license_file: 
+  summary: "User-friendly command-line interface for the ZARP RNA-Seq analysis pipeline"
+  doc_url: https://zavolanlab.github.io/zarp-cli 
+  dev_url: https://github.com/zavolanlab/zarp-cli/blob/dev 
```

Comment:
This should be `https://github.com/zavolanlab/zarp-cli/tree/dev` (`tree` instead of `blob`). Sorry about that.